src/styles/blocks/catalog.scss

```
@@ -1,17 +1,16 @@
 .catalog {
   display: flex;
   flex-wrap: wrap;
-
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
+  transition-property: 300ms, color, transform;
```

Your transition doesn't work. And move 300ms to a variable, as I said before
```suggestion
  transition-property: color $transition-duration, transform $transition-duration;
```